angular-workspace/projects/ni/nimble-angular/rich-text-editor/testing/rich-text-editor.pageobject.ts

```diff
@@ -0,0 +1,3 @@
+import { RichTextEditorPageObject } from '@ni/nimble-components/dist/esm/rich-text-editor/testing/rich-text-editor.pageobject';
+
+export { RichTextEditorPageObject };
```

Comment:
think we also need to `export type { ToolbarButton }` from `@ni/nimble-components/dist/esm/rich-text-editor/testing/types` which is used by the page  object

Reply:
Thanks for pointing it out. I have updated the same.